internal/bminventory/inventory.go

```diff
@@ -2037,6 +2037,10 @@ func (b *bareMetalInventory) updateNetworkParams(params installer.UpdateClusterP
 		setMachineNetworkCIDRForUpdate(updates, machineCidr)
 	}
 	if params.ClusterUpdateParams.NetworkType != nil && params.ClusterUpdateParams.NetworkType != cluster.NetworkType {
+		// If user
+		b.setUsage(true, usage.NetworkTypeSelectionUsage, &map[string]interface{}{
+			"network_type": params.ClusterUpdateParams.NetworkType}, usages)
+
 		updates["network_type"] = swag.StringValue(params.ClusterUpdateParams.NetworkType)
```

Comment:
is there a default network_type? If yes it should be added to setDefaultUsage() as well